lib/accesslint/ci/cli.rb

```diff
@@ -0,0 +1,17 @@
+require "thor"
+
+module Accesslint
+  module Ci
+    class CLI < Thor
```

Comment:
Just realized what you meant George. I think you're right, I changed `CLI` to `Cli`.